pages/sites/setting/environments.ftd

```diff
@@ -0,0 +1,160 @@
+-- import: ui.fifthtry.com/components/sites/setting/page as p
+-- import: ui.fifthtry.com/site-data
+-- import: ui.fifthtry.com/records/user-data as ud
+-- import: ui.fifthtry.com/records/sites as sites-records
+-- import: ui.fifthtry.com/actions/sites/settings/environment
+
+
+-- component page:
+module site-data: site-data
+module action: environment
+sites-records.Environment list envs:
```

Comment:
This should be lowercases environment. We do not use capital case to define records etc. At some point we can review this convention, but for now since this is the prevailing convention please follow here as well.

Reply:
the capital Environment was visually easier to identify for me but I agree, sticking to conventions is the right thing.

fixed in https://github.com/FifthTry/ui/pull/58/commits/ddbda885d4f3f4085989b03d004b33719b6720a5

PS: We should totally revisit the https://fastn.com/geeks/ page and rethink our conventions. I personally don't like the indented formatting of ftd files via `fastn fmt`, I think [code folding](https://en.wikipedia.org/wiki/Code_folding#:~:text=Code%20folding%20allows%20one%20to,allowing%20a%20summary%2Fdetailed%20view) through editor tooling would be much better to reduce the clutter. FWIW my opinion also aligns with the /geeks/ page.